SurveyApp/Services/Network/API/APIResponseSerializer.swift

```diff
@@ -43,6 +40,9 @@ final class APIResponseSerializer<T: Decodable>: ResponseSerializer {
             if response.statusCode > 300 {
                 let result = try errorSerializer.serialize(request: request, response: response, data: data, error: nil)
                 return .failure(result)
+            } else if data == " ".data(using: .utf8) {
+                let result = try successSerializer.serialize(request: request, response: response, data: emptyData, error: nil)
+                return .success(result)
             } else {
                 let result = try successSerializer.serialize(request: request, response: response, data: data, error: nil)
                 return .success(result)
```

Comment:
Let make it more readable by removing `else if` statement 👍 
```suggestion
            if response.statusCode > 300 {
                let result = try errorSerializer.serialize(request: request, response: response, data: data, error: nil)
                return .failure(result)
            }
            
            if data == " ".data(using: .utf8) {
                let result = try successSerializer.serialize(request: request, response: response, data: emptyData, error: nil)
                return .success(result)
            }
            
            let result = try successSerializer.serialize(request: request, response: response, data: data, error: nil)
            return .success(result)
```

Reply:
I updated the code. Thank you for pointing that out 🎯 🙏